src/host/exe/Host.EXE.vcxproj

```diff
@@ -86,7 +86,7 @@
     </ClCompile>
     <Link>
       <AllowIsolation>true</AllowIsolation>
-      <AdditionalDependencies>WinMM.Lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>winmm.lib;imm32.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
Which API do we need from this? If it is part of an APIset, we should prefer apiset linkage. We may also need to update `sources`.